wiki/People/The_Team/pl.md

```diff
@@ -1,50 +1,51 @@
----
-outdated: true
----
-
 Załoga
 ========
 
+Wymienione poniżej osoby są rdzeniem zespołu Osu i głównie odpowiadają za pomoc w sprawianiu, by gra była wspaniała dla wszystkich.
+
 Twórca
 ----------------------
 
 | Pseudonim | Opis |
 | ---- | ----------- |
-| [peppy](https://osu.ppy.sh/u/2) | Prowadzenie projektu |
+| [![][flag_AU] peppy](https://osu.ppy.sh/u/2) | Prowadzenie projektu |
 
 Aktywni
 ----------------------
 
 | Pseudonim | Opis |
 | ---- | ----------- |
-| [Ephemeral](https://osu.ppy.sh/u/102335) | Członek supportu, Sekretarz Meganekko |
-| [flyte](https://osu.ppy.sh/u/3103765), [osu!next](http://next.ppy.sh/) | Główny grafik |
-| [Loctav](https://osu.ppy.sh/u/71366) | Organizowanie oficjalnych turniejów, Zarządzanie staffem i społecznością |
-| [nanaya](https://osu.ppy.sh/u/2387883) | Realizacja strony |
-| [RBRat3](https://osu.ppy.sh/u/307202) | Rysownik osu!, Robienie odjechanych rzeczy |
-| [smoogipooo](https://osu.ppy.sh/u/1040328) | Twórca osu!manii, Naprawianie bugów |
-| [Tom94](https://osu.ppy.sh/u/1857058) | Twórca pp |
-| [yelle](https://osu.ppy.sh/u/4916903) | Zarządzanie sklepem osu! |
-| [ztrot](https://osu.ppy.sh/u/6347) | Profesor osu!academy, wygląd maskotek |
-| [Zallius](https://osu.ppy.sh/u/55) | Zagrożony wyginięciem |
+| [![][flag_NO] cYsmix](https://osu.ppy.sh/u/272870) | Audio design & [Wyróżniony artysta](https://osu.ppy.sh/beatmaps/artists/2) |
+| [![][flag_AU] Ephemeral](https://osu.ppy.sh/u/102335) | Media, Członek supportu, Sekretarz Meganekko |
+| [![][flag_JP] flyte](https://osu.ppy.sh/u/3103765) | Główny grafik, [osu!next](http://next.ppy.sh/) |
+| [![][flag_JP] nanaya](https://osu.ppy.sh/u/2387883)) | Realizacja strony |
+| [![][flag_AU] nekodex](https://osu.ppy.sh/u/102) | Realizacja strony |
+| [![][flag_US] RBRat3](https://osu.ppy.sh/u/307202) | Rysownik osu!, Robienie odjechanych rzeczy |
+| [![][flag_FR] Shiro](https://osu.ppy.sh/u/113005) | Zarządzanie społecznością |
```

Comment:
w angielskiej wersji masz *Player support*, co to ma wspólnego ze społecznością? powinno być coś w rodzaju *wsparcie techniczne* (*player support* -> *wsparcie dla graczy* brzmi dziwnie)